<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNewVisitor</class>
 <widget class="QWidget" name="addNewVisitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25,43,63)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QLineEdit" name="middle_name_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);
</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="first_name_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);
</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ffffff;&quot;&gt;Заселение гостя&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="passport_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Фамилия&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Имя&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Отчество&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Номер паспорта&lt;/span&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="last_name_line">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Количество гостей&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_11">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Длительность &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;проживания&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Город&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="duration">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>365</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="city">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_14">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot;white-space: pre-wrap; font-weight:600; color: white;&quot;&gt;Номер    Размер    Стоимость&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="rooms">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="statusTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="accessibleName">
          <string/>
         </property>
         <property name="accessibleDescription">
          <string>лпдт</string>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 57, 84);
color: rgb(255, 255, 255);
selection-background-color: rgb(65, 116, 167);
selection-color: rgb(25, 44, 64);
border-color: rgb(25,43,63);</string>
         </property>
         <property name="text">
          <string>Готово</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name_line</tabstop>
  <tabstop>middle_name_line</tabstop>
  <tabstop>last_name_line</tabstop>
  <tabstop>passport_line</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>duration</tabstop>
  <tabstop>city</tabstop>
  <tabstop>rooms</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
